zc_install/ajaxAdminSetup.php

```diff
@@ -12,7 +12,7 @@
 
 require(DIR_FS_INSTALL . 'includes/application_top.php');
 
-$adminDir = $_POST['adminDir'];
+$adminDir = $_POST['adminDir'] ?? 'admin';
```

Comment:
?? was introduced in php 7 
Assuming 1.5.8 will have a minimum php version of 7 then fine. If not will need `$adminDir = isset($_POST['adminDir']) ? $_POST['adminDir'] : 'admin';`

Reply:
Ingore. Have just read the whats new in 1.5.8 says php min version 7.3.